packages/blockchain-wallet-v4-frontend/src/scenes/Home/Banners/RewardsBanner/index.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { PureComponent } from 'react'
+import { FormattedMessage } from 'react-intl'
+import { connect, ConnectedProps } from 'react-redux'
+import { LinkContainer } from 'react-router-bootstrap'
+import { bindActionCreators } from '@reduxjs/toolkit'
+import styled from 'styled-components'
+
+import { Icon, Text } from 'blockchain-info-components'
+import { actions } from 'data'
+import { Analytics } from 'data/types'
+import { media } from 'services/styles'
+
+import { BannerButton, IconWrapper } from '../styles'
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  border: 1px solid ${(props) => props.theme.grey000};
+  border-radius: 8px;
+  padding: 20px;
+
+  ${media.atLeastLaptop`
+    height: 80px;
+    padding: 0 20px;
+  `}
+  ${media.mobile`
+    padding: 12px;
+    flex-direction: column;
+  `}
+`
+const Row = styled.div`
```

Comment:
You could use Flex Component instead of adding this Row and Column styling
e.x.: 
`<Flex alignItems='center'>`

Reply:
Is not in constellation. It is in the component package of the wallet

Reply:
Ohhh will use it next time :)